test/Microsoft.ML.Functional.Tests/ModelLoading.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace Microsoft.ML.Functional.Tests
 {
-    public partial class ModelLoadingTests : BaseTestClass
+    public partial class ModelLoadingTests : TestDataPipeBase
     {
         private MLContext _ml;
```

Comment:
>_ml [](start = 26, length = 3)

`TestDataPipeBase` may have an `MLContext` field.

Reply:
Ah good point thanks.

Reply:
I will have two commits in my next push -- one to resolve other comments, and one to do this rename, since it winds up being a little annoying and distracting in this file.

Reply:
Actually, no sorry, I don't have time to do this right now and need to run to another meeting, so I'll just leave as is and fix later (since I think other more "public facing" changes might be blocking the PR more than this).

Reply:
OK! I have finally done this, sorry for not doing this earlier @yaeldekel but it was one of these situations where i had literally 30 seconds before I had to go somewhere and the change would take me 1 minute. 😄 But I just got that 1 minute.